upath/implementations/local.py

```diff
@@ -63,6 +63,22 @@ def _from_parts(cls, args, *, url=None, **kw):
         obj._url = SplitResult("", "", str(obj), "", "")
         return obj
 
+    @classmethod
+    def _from_parsed_parts(
+        cls,
+        drv,
+        root,
+        parts,
+        url=None,
+        **kwargs: Any,
+    ):
+        obj = super(UPath, cls)._from_parsed_parts(  # type: ignore[misc]
```

Comment:
I am not sure why `mypy` complained here.

Reply:
Not sure either. Especially since it didn't complain with `_from_parts`...